test/zzz-duplicate-entries.sh

```diff
@@ -49,10 +49,12 @@ wait_for_fluentd_to_catch_up
 # dump random messages into journal for the next 35 seconds to ensure that we've rolled over
 MESSAGES=""
 TIMES=0
+opslogline=$( mktemp )
 while [ $TIMES -lt 35 ]; do
   MESSAGE="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   MESSAGES="$MESSAGES $MESSAGE"
-  add_test_message $MESSAGE
+  echo -e $MESSAGE > opslogline
+  logger -i -p local6.info -t $MESSAGE -f $opslogline
```

Comment:
so yes, a little different use case than the bulk rejection test.  In your case, you want systemd.u.SYSLOG_IDENTIFIER to be the same as the message field:
```
 while [ $TIMES -lt 35 ]; do
   MESSAGE="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   logger -i -p local6.info -t $MESSAGE $MESSAGE
   let TIMES=TIMES+1
 done
```
Then you don't need opslogline at all

Reply:
awesome, I wasn't sure if that was required or not... I'll update to remove the `-f` and just use `$MESSAGE`